.github/workflows/ci.yaml

```diff
@@ -1,6 +1,10 @@
 name: Dotty CI
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 3 * * *'  # Every day at 3 AM
```

Comment:
3 AM on which time zone?

Reply:
UTC. From https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule:

> You can schedule a workflow to run at specific UTC times using POSIX cron syntax.